pandas/core/arrays/datetimelike.py

```diff
@@ -1518,6 +1518,30 @@ def mean(self, skipna=True):
         return self._box_func(result)
 
 
+def is_valid_na(obj, dtype):
```

Comment:
what??? yet another missing value check. This should not be here at all.

Reply:
i guess it could go in _libs.missing?